nx.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "node_modules/nx/schemas/nx-schema.json",
+  "namedInputs": {
+    "default": ["{projectRoot}/**/*"],
+    "prod": ["!{projectRoot}/test/**/*"]
+  },
+  "targetDefaults": {
+    "build": {
+      "dependsOn": ["^build"],
+      "inputs": ["prod", "^prod"],
+      "outputs": ["{projectRoot}/lib", "{projectRoot}/types"]
+    },
+    "test-typings": {
+      "inputs": ["prod", "^prod"]
```

Comment:
does this mean it's not going to run typing tests if test files have changed?

Reply:
Good point. I've updated this now to be the same as `test-unit`.

The `test-unit` and `test-typings` scripts will now run when any file changes in the package or when a non-test file in a related package changes.